src/assets/helpers/youtubeUrlFormatter.ts

```diff
@@ -0,0 +1,13 @@
+export function youtubeUrlFormatter(url:string):string{
+    const actualUrl = new URL(url)
+    const validateUrl = /^(https?\:\/\/)?(www\.youtube\.com|youtu\.be)\/.+$/
+    const baseUrl = 'https://www.youtube-nocookie.com/embed/'
+
+    if(validateUrl.test(actualUrl.href)){
+        const href = actualUrl.href.replace('watch?v=', '')
```

Comment:
aqui podemos ter problemas com url curta do youtube

Reply:
refatorado, formatos de url testados:

https://youtu.be/YD5hBvZ9fFo
https://youtu.be/embed/YD5hBvZ9fFo
https://youtu.be/YD5hBvZ9fFo/&y=11
https://www.youtube.com/watch?v=YD5hBvZ9fFo&feature=youtu.be
https://www.youtube.com/watch?v=YD5hBvZ9fFo?fs=1&hl=en_US
https://www.youtube.com/watch?v=YD5hBvZ9fFo
https://www.youtube.com/embed/YD5hBvZ9fFo
https://www.youtube-nocookie.com/embed/DFYRQ_zQ-gk?autoplay=1
https://www.youtube-nocookie.com/watch?v=YD5hBvZ9fFo